packages/grafana-toolkit/src/cli/tasks/plugin.build.ts

```diff
@@ -66,6 +66,7 @@ export const lintPlugin = useSpinner<Fixable>('Linting', async ({ fix }) => {
   try {
     // Show a warning if the tslint file exists
     await fs.access(resolvePath(process.cwd(), 'tslint.json'));
+    console.log('\n');
```

Comment:
Kinda unrelated, but cleanup while we are looking...  without this extra newline the first warning shows up at the end of other test